app/client/views/admin/user/user.html

```diff
@@ -149,8 +145,8 @@
       <div class="field"
            ng-hide="isMitStudent">
         <p>
-          Because of limitations imposed by MIT, we are not legally allowed to host
-          non-WIT minors (those under 18) for HackWITus 2015. Checking the box below affirms that you are or will be 18 years or older by September 19th, 2015.
+          Because of limitations imposed by WIT, we are not legally allowed to host
+          non-WIT minors (those under 18) for HackWITus 2019. Checking the box below affirms that you are or will be 18 years or older by September 19th, 2015.
```

Comment:
Change `September 19th, 2015` to `November 9th, 2019`

Reply:
This is the admin side of things, It doesn't really matter.

Reply:
Ah okay